docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -132,3 +178,19 @@ $ open -a "Teleport Connect" --args --insecure
 # Passing the flag to the executable directly:
 $ /Applications/Teleport\ Connect.app/Contents/MacOS/Teleport\ Connect --insecure
 ```
+</TabItem>
+<TabItem label="Linux">
+From a terminal, open Teleport Connect with the `--insecure` flag:
+
+```code
+$ teleport-connect --insecure
+````
+</TabItem>
+<TabItem label="Windows">
+From the Command prompt, open Teleport Connect with the `--insecure` flag:
+
+```code
+$ "C:\Users\yourUser\AppData\Local\Programs\teleport-connect\Teleport Connect.exe" --insecure
```

Comment:
The downside of this is that the user cannot just copy and paste it because they need to replace `yourUser` with their username.

I looked up if we can improve it and it turns out we can.

```suggestion
$ "%LocalAppData%\Programs\teleport-connect\Teleport Connect.exe" --insecure
```

https://superuser.com/questions/681744/how-to-generically-refer-to-the-appdata-folder-on-the-windows-command-line